### PR TITLE
Add iOS quick start page

### PR DESCRIPTION
**Pull request by cbush**
- Seems to be missing a relationship example.
- Rewrap is not working for some reason. Will do a pass later.

## JIRA 
https://jira.mongodb.org/browse/DOCSP-10332

## Staged
https://docs-mongodbcom-staging.corp.mongodb.com/realm/bush/quick-start/ios/quick-start.html

### CODE REVIEW

File: source/ios/quick-start.txt [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+This page contains everything you need to know to get {+client-database+}
```
Comment (terakilobyte) [inline source/ios/quick-start.txt:18]:
```suggestion
This page contains information to quickly get {+client-database+}
```

File: source/ios/quick-start.txt [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+statement:
+
+.. code-block:: swift
+
+   import RealmSwift
+
+Initialize the App
+------------------
+
+If you are using {+backend+} features like authentication and sync, you must
```
Comment (terakilobyte) [inline source/ios/quick-start.txt:38]:
```suggestion
To use {+backend+} features such as authentication and sync, you must
```

File: source/ios/quick-start.txt [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+access your {+app+} using your {+app+} ID. You can find your {+app+} ID in the
+{+ui+}.
```
Comment (terakilobyte) [inline source/ios/quick-start.txt:40]: Link here or provide a screenshot

File: source/ios/quick-start.txt [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+Define Your Object Model
+------------------------
+
+If have not enabled {+sync+} or you enabled {+sync-short+} with :term:`Developer
+Mode` in the {+ui+}, you can define your :ref:`object model <ios-realm-objects>`
+directly in code.
+
+.. note::
+
+   If you have enabled {+sync-short+} but turned off Developer Mode, you can copy and paste the object model definitions that {+backend-short+} generated for you from the :guilabel:`SDKs` tab in the {+ui+}. You must re-enable Developer Mode if you want to make changes to the object model definition from client side code. See :ref:`Configure Your Data Model <configure-your-data-model>`.
```
Comment (terakilobyte) [inline source/ios/quick-start.txt:63]: I think a screenshot here would be invaluable

File: source/ios/quick-start.txt [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+
+.. code-block:: swift
+
+   class Task: Object {
+       @objc dynamic var _id: ObjectId = ObjectId.generate()
```
Comment (terakilobyte) [inline source/ios/quick-start.txt:68]: Is _id and _partition required for every realm object now, or only those being synced to MongoDB? May want to call this out in a comment.

File: source/ios/quick-start.txt [hunk 6]
```diff
@@ -0,0 +1,240 @@
...
+           self.init()
+           self._partition = partition
+           self.name = name
+       }
+   }
+
+Authenticate a User
+-------------------
+
+If you have enabled :ref:`anonymous authentication <anonymous-authentication>`, you can log in anonymously to your app:
```
Comment (terakilobyte) [inline source/ios/quick-start.txt:87]:
```suggestion
Anonymous Authentication
~~~~~~~~~~~~~~~~~~~

Enabling :ref:`anonymous authentication <anonymous-authentication>` allows users to immediately log into your app.
```

File: source/ios/quick-start.txt [hunk 7]
```diff
@@ -0,0 +1,240 @@
...
+
+          // Now logged in, do something with user
+
+        }
+    }
+
+
+If you have enabled :ref:`email/password authentication
+<email-password-authentication>`, you can register a new user with an email and
+password:
```
Comment (terakilobyte) [inline source/ios/quick-start.txt:108]:
```suggestion
Email/Password
~~~~~~~~~~~

With :ref:`email/password authentication
<email-password-authentication>`, you can register a new user with an email and
password.
```

File: source/ios/quick-start.txt [hunk 8]
```diff
@@ -0,0 +1,240 @@
...
+
+.. code-block:: swift
+
+   app.usernamePasswordProviderClient().registerEmail(username, password: password, completion: { (error) in
+     // Handle errors...
+     // Registering just registers. You can now log in.
+   }
+
+If you have enabled :ref:`email/password authentication
+<email-password-authentication>`, you can log in with email and password:
```
Comment (terakilobyte) [inline source/ios/quick-start.txt:118]:
```suggestion
And you can log in with:
```

File: source/ios/quick-start.txt (hunk 6 above)
Comment (cbush) [reply source/ios/quick-start.txt]: Reworked in the vein of suggestion

File: source/ios/quick-start.txt (hunk 3 above)
Comment (cbush) [reply source/ios/quick-start.txt:40]: I'm even thinking we could use a very brief but central page for this. We are going to say "Find your App ID" a lot!

**Pull request merged**